pkgs/applications/networking/deck/default.nix

```diff
@@ -23,6 +23,8 @@ in buildGoModule rec {
 
   vendorHash = "sha256-jhLZvusYpX5fW1NCmJtwE/p9/wTwzA2hbwt657VsZts=";
 
+  passthru.updateScript = ./update.sh;
```

Comment:
Try `nix-update-script`, also we don't really need the `short_hash` thing (it should match the tag?).

Reply:
How do you use `nix-update-script`?  I searched around and hardly find anything useful.

I know it is not required, I keep this `short_hash` just to match the official release as possible.

Reply:
Just set `passthru.updateScript = nix-update-script { };` and everything magically works.

Reply:
> Just set `passthru.updateScript = nix-update-script { };` and everything magically works.
😲 
Just want to clarify, do I still need  `update.sh` in the same folder? If not then how does nix-update knows to pull `short_hash` in the new version?

A little bit of background info.

The reason for using  `passthru.updateScript` is to solve the automate update process does not update `short_hash`. https://github.com/NixOS/nixpkgs/pull/248291

Reply:
No it does not do that. You could use nix-update to pull and update the latest version in your script, which saves a few lines of code. But maybe not worth the hassle given that you already wrote the code.

Reply:
ok. Looks like the update script didn't actually work as of now.

Reply:
> The reason for using `passthru.updateScript` is to solve the automate update process does not update `short_hash`. #248291

That's why I said that we don't really need `short_hash` and can just drop it instead.

Reply:
I think so too.

Reply:
I am curious to learn what's the general practice for maintaining a nixpkges for NixOS 

Let's say when there is no compatibility issues, do we do what is minimum to build the package or do we try to build the package as close to official ones as possible?